templates/faq.html

```diff
@@ -98,7 +99,7 @@ <h1>שאלות נפוצות</h1>
 </body>
 <footer>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
-    <script src="js/semantle.js?version={{js_version}}"></script>
+    <script src="//static/js/semantle.js?version={{js_version}}"></script>
```

Comment:
this doesn't work - here and elsewhere

Reply:
c87f89be6451ce4e7e5f810b019adb7b38badd2f

Reply:
without this, when going to 127.0.0.1:8000/faq, I get a request for 127.0.0.1:8000/faq/static/semantle.js?version={{js_version}}